.gitignore

```diff
@@ -0,0 +1 @@
+/test.md
```

Comment:
**[REQUIFED CHANGES]**

- Kindly add the `.hintrc`, and `.stylelintrc.json` linters files in your root folder from the [microverse linter](https://github.com/microverseinc/linters-config/tree/master/html-css) repository.
- Kindly add the `node_modules/` directory in your `.gitignore file`